src/fetch.js

```diff
@@ -297,15 +297,15 @@ const getSyncData = async (
     */
     const aggregatedSyncToken = syncToken.filter(item => item !== undefined);
       for (const token of aggregatedSyncToken) {
-
+        let SyncRetryCount;
         let syncResponse;
         try {
-          
+          SyncRetryCount = 0;
           syncResponse = await fetchCsData(
           url,
           config,
             (query = { sync_token: token }),
-          0 // Reset SyncRetryCount for each call
+            SyncRetryCount // Reset SyncRetryCount for each call
```

Comment:
can we initialize the SyncRetryCount outside the loop and handle retrycount in getSyncedData function ? As what i can understand the SyncRetryCount is not used anywhere in fetchCsData function which is redundant. 
To persist the SyncRetryCount as retry count is initialized to 0 in each iteration.